#ifndef VENTANA_H
#define VENTANA_H

#include <QWidget>

namespace Ui {
class Ventana;
}

class Ventana : public QWidget
{
    Q_OBJECT

public:
    explicit Ventana(QWidget *parent = nullptr);
    ~Ventana();

protected:
    void paintEvent(QPaintEvent *event) override;

private slots:
    void slot_botonClic();  // <--- AGREGALO ACA

private:
    Ui::Ventana *ui;
};

#endif // VENTANA_H
